{
  "detail": "Request failed, see errors for additional details.",
  "io.fusionauth.errors": {
    "generalErrors": [
      {
        "code": "[invalid]",
        "message": "Your JSON was invalid"
      }
    ],
    "fieldErrors": {
      "user.username": [
        {
          "code": "[duplicate]user.username",
          "message": "A User with username = [neo] already exists."
        }
      ]
    }
  },
  "schemas": [
    "io.fusionauth.errors",
    "urn:ietf:params:scim:api:messages:2.0:Error"
  ],
  "status": "400"
}